### DISCUSSION TITLE
Why console.log output doesn't show in xcode?

### DISCUSSION THREAD

**Discussion by shaifulborhan** [ionic-team/capacitor#7791] [Q&A]
I am using Ionic 8 + Capacitor 6 in a fresh project. Why does the console.log output not showing in the native xcode console? Has something changed? I remember this was not the case in previous Capacitor versions.

I tried to follow the instructions in the [docs](https://capacitorjs.com/docs/vscode/debugging#use-safari) to use Safari but it couldn't detect the iOS app in the simulator and showing the "No inspectable applications" message.

My workflow is:
1. ionic cap sync ios
2. ionic cap open ios
3. Run app in iOS 17.0 iPhone simulator

Thank you.

**Comment by jcesarmobile** [comment]
Are you using the CocoaPods project (default), or the SPM project?

**Reply by shaifulborhan** [reply]
@jcesarmobile SPM.

**Reply by jcesarmobile** [reply]
yeah, that's a limitation of SPM since Capacitor there is distributed as a precompiled xcframework, so the `DEBUG` compiler flag doesn't work there and by default Capacitor won't output logs if `DEBUG` is false, which is for SPM, so if you want logs you have to enable them for production, and also have to explicitly set the WebView as debuggable.
Check `loggingBehavior` and `webContentsDebuggingEnabled` configuration options 
https://capacitorjs.com/docs/config
